src/components/macro/macro-action-editor/macro-action-editor.component.ts

```diff
@@ -9,7 +9,7 @@ import {MacroTextTabComponent} from './tab/macro-text/macro-text.component';
 import {MacroDelayTabComponent} from './tab/macro-delay/macro-delay.component';
 import {MacroMouseTabComponent} from './tab/macro-mouse/macro-mouse.component';
 
-import {MacroItemComponent} from '../../popover/tab/macro/macro-item.component';
+import {MacroItemComponent} from '../macro-item/macro-item.component';
```

Comment:
Create an index.ts in the macro-item folder export the MacoItemComponent from there and use '../macro-item' here as a path.